extensionsGallery.json

```diff
@@ -63,6 +63,59 @@
 					"statistics": [],
 					"flags": "preview"
 				},
+				{
+					"extensionId": "10",
```

Comment:
I'm not sure what the extensionId is

Reply:
It's just an identifier. Use 57 for this and then make sure to update TotalCount to 47 below

Reply:
got it, thanks. I will also move this to the bottom of the JSON